src/armTemplates/resources/functionApp.ts

```diff
@@ -78,45 +92,14 @@ export class FunctionAppResource implements ArmResourceTemplateGenerator {
             "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]",
             "[concat('microsoft.insights/components/', parameters('appInsightsName'))]"
           ],
-          "kind": "functionapp",
+          "kind": "[parameters('functionAppKind')]",
           "properties": {
             "siteConfig": {
-              "appSettings": [
```

Comment:
Where do tha appSettings get set if its not in the ARM template?  Making an az cli command after now?

Reply:
The logic for which app settings are set is now encapsulated in its own function. (line 98 below this comment)